src/Http/Request.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+namespace Framework3\Http;
+
+use Framework3\Http\Bag\ServerBag;
+use Framework3\Http\Bag\GetBag;
+use Framework3\Http\Bag\PostBag;
+use Framework3\Http\Bag\CookieBag;
+use Framework3\Http\Bag\FilesBag;
+use Framework3\Bag\Bag;
+use Framework3\Http\Session\SessionInterface;
+
+class Request
+{
+    public const METHOD_GET =       'GET';
+    public const METHOD_HEAD =      'HEAD';
+    public const METHOD_POST =      'POST';
```

Comment:
Codacy found an issue: [Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6299285)